.github/workflows/checkXSDValidity.yml

```diff
@@ -0,0 +1,17 @@
+name: Check XSD validity
+on:
+  schedule:
+    #  * is a special character in YAML so you have to quote this string
+    - cron: "0 0 1,7,14,21,28 * *"
```

Comment:
Every week is fine!
Is "0 0 * * 0" not roughly doing the same (every Sunday at 00:00?
Maybe more clean :)

Reply:
Effectively, it's more clear. I will update it.

Reply:
Nice!